src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubOptions.cs

```diff
@@ -60,6 +61,8 @@ public int BatchCheckpointFrequency
             }
         }
 
+        public InitialOffsetOptions InitialOffsetOptions { get; set; }
```

Comment:
Add these options to `public string Format()`

Reply:
Good catch!  Added.